mvp-spec/workflows.md

```diff
@@ -0,0 +1,84 @@
+# Workflows
+
+The workflows part of the Prebid Addressability Framework are:
+- User Id and Preferences
+- Selling ad slots
+
+## Id and User Preferences
+
+### Generic workflow
+
+This workflow illustrates retrieving and creating/setting User Id and Preferences.
+
+```mermaid
+sequenceDiagram
+participant U as User
+participant P as Participant's website
+participant O as Operator
+
+    U ->> P: visit site
+    
+    P ->> O: Read User Id and Preferences
+    O ->> P: No User Id and Preferences
+    
+    P ->> U: Display consent prompt
+    U ->> P: Give consent
+
+    P ->> O: Write User Id and Preferences
+    O ->> O: Generate<br />User Id and Preferences
+    O ->> P: Forward User Id and Preferences
+```
+### When third-party cookies are supported
+
+When third-party cookies are supported in the browser:
+- User Id and Preferences are stored on the Operator's domain and retrieved in third-party context from the participant's website.
```

Comment:
Actually, this is not correct: in both cases (3PC or not), the cookies are replicated locally, to avoid having to do extra calls to the operator.
The difference is only in the way they are initially requested from the operator, or when they need to be refreshed (because the local copy is out of date)

Reply:
Ok, new commit updating this pragraph.